orbit/pkg/packaging/packaging.go

```diff
@@ -122,6 +122,9 @@ type Options struct {
 	EndUserEmail string
 	// DisableKeystore disables the use of the keychain on macOS and Credentials Manager on Windows
 	DisableKeystore bool
+	// OsqueryDB is the directory to use for the osquery database.
+	// If not set, then the default is `$ORBIT_ROOT_DIR/osquery.db`.
+	OsqueryDB string
```

Comment:
Why not `OsqueryDBPath` ?

Reply:
To keep it consistent with the flag name `--osquery-db`.